packages/iris-grid/src/mousehandlers/IrisGridContextMenuHandler.tsx

```diff
@@ -717,44 +717,56 @@ class IrisGridContextMenuHandler extends GridMouseHandler {
 
     const actions: ResolvableContextAction[] = [];
 
-    if (modelColumn != null && modelRow != null) {
-      const value = model.valueForCell(modelColumn, modelRow);
-
-      const valueText = model.textForCell(modelColumn, modelRow);
-      const column = columns[modelColumn];
+    if (modelColumn != null) {
+      const sourceCell = model.sourceForCell(modelColumn, modelRow ?? 0);
+      const { column: sourceColumn, row: sourceRow } = sourceCell;
+      if (modelRow != null) {
```

Comment:
Why change this from checking both row and column are not null to just column, defaulting to 0 row, then checking row is not null?

Reply:
Yea I had this wired up differently previously before I added the `getClearFilterRange` method. I'll change it back, as this change doesn't really make sense in this context.